engine/app/components/citizens_advice_components/form_group.html.erb

```diff
@@ -1,29 +1,31 @@
 <div class="cads-form-field<% if error? %> cads-form-field--has-error<% end %>">
   <% if error? %><div class="cads-form-field__error-marker"></div><% end %>
-  <%= tag.fieldset(class: fieldset_classes) do %>
-    <legend class="cads-form-field__label<% if legend_heading? %> cads-form-field__label--with-heading<% end %>">
-      <% if legend_heading? %>
-        <h1 class="cads-form-field__heading"><%= legend %></h1>
-      <% else %>
-        <%= legend %>
+  <div class="cads-form-field__content">
+    <%= tag.fieldset(class: fieldset_classes) do %>
+      <legend class="cads-form-field__label">
+        <% if page_heading? %>
+          <h1 class="cads-page-title"><%= legend %></h1>
+        <% else %>
+          <%= legend %>
+          <% if optional? %>
+            <span class="cads-form-field__optional"><%= "(#{t('citizens_advice_components.input.optional')})" %></span>
+          <% end %>
```

Comment:
For consistency with other fields a field using `page_heading` can't use the optional marker as they are contradictory goals.